ecs_fargate/README.md

```diff
@@ -8,65 +8,92 @@ Get metrics from all your containers running in ECS Fargate:
 * I/O metrics
 
 ## Setup
-### Installation
-
-The ECS Fargate check is packaged with the Agent, [run the Agent][1] with your containers to start collecting metrics.
-
-### Deploying the Datadog Agent on AWS Fargate
+The following steps cover setup of the Datadog Container Agent within AWS ECS Fargate. **Note**: Datadog Agent version 6.1.1 or higher is needed to take full advantage of the Fargate integration.
 
-To start monitoring AWS Fargate tasks and services, deploy the containerized Datadog Agent on Fargate. In addition to passing the usual `DD_API_KEY` environment variable, you must set the `ECS_FARGATE `environment variable to `true`.
+### Installation
+To monitor your ECS Fargate containers with Datadog, run the Agent as a container in same task definition as your application. To collect metrics with Datadog, each task definition requires a Datadog Agent container. Follow these setup steps:
 
-The example task definition below deploys the Datadog Agent to Fargate, along with a Redis container in the same task.
+1. **Add an ECS Fargate Task**
+2. **Create or Modify your IAM Policy**
+3. **Run the Datadog Agent as a Replica Service**
 
-```
-{'containerDefinitions': [{'dockerLabels': {'com.datadoghq.ad.check_names': '["redisdb"]',
-                                            'com.datadoghq.ad.init_configs': '[{}]',
-                                            'com.datadoghq.ad.instances': '[{"host": "%%host%%", "port": 6379}]'},
-                           'essential': true,
-                           'image': 'redis:latest',
-                           'name': 'redis'},
-                          {'environment': [{'name': 'DD_API_KEY',
-                                            'value': '$YOUR_API_KEY'},
-                                           {'name': 'ECS_FARGATE',
-                                            'value': 'true'}],
-                           'essential': true,
-                           'image': 'datadog/agent:latest',
-                           'name': 'datadog-agent'}],
- 'cpu': '256',
- 'family': 'redis-datadog',
- 'memory': '512',
- 'networkMode': 'awsvpc',
- 'requiresCompatibilities': ['FARGATE']}
-```
+#### Create an ECS Fargate Task
+This task launches the Datadog Agent container. Configure the task using the [AWS CLI tools][11] or the [Amazon Web Console][12].
```

Comment:
Are we intentionally dropping the example task definition?

Reply:
Also as a reminder this isn't its own dedicated task, instead Datadog needs to run inside each application task the customer wants to monitor. They should never deploy a task with JUST datadog in it..

Reply:
^ This. I liked previous the agent + redis task example because it shows how to combine the agent and the application to monitor.

Reply:
The example is in a separate doc linked here:
https://github.com/DataDog/integrations-core/blob/ruth/ecs-fargate-docs/ecs_fargate/README.md#metric-collection

Link to separate doc:
https://docs.datadoghq.com/integrations/faq/integration-setup-ecs-fargate/

Additionally, I will add a note to this section to make it clearer additional items need to be added to the same task as the Datadog Agent.